### PR TITLE
Merge dev to main

### PR DESCRIPTION
**Pull request by gageway1**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_HOST: db
+    networks:
+      - ciabot_network
+    depends_on:
+      - db
+    ports:
+      - "81:3001"
```
Comment (isthistechsupport) [inline docker-compose.yml:32]: What's the port for?

**Review by isthistechsupport — COMMENTED** [review]
LGTM but I want to change list separation before approval

File: docker-compose.yml (hunk 1 above)
Comment (gageway1) [reply docker-compose.yml:32]: port is for the bot itself, i just incremented one up of the default of 80/3000.

**Comment by isthistechsupport** [comment]
Ok list separation added

File: docker-compose.yml (hunk 1 above)
Comment (isthistechsupport) [reply docker-compose.yml:32]: okay I might look into it later since I'm not sure if we need to expose container ports atm

**Review by isthistechsupport — APPROVED** [review]
LGTM

**Pull request merged**